readable-stream-test/runner-prepare.mjs

```diff
@@ -0,0 +1,77 @@
+import { exec } from 'child_process'
```

Comment:
why is this needed?

Reply:
See line 27 (note this is a copy /paste of a subset of the code in the nodejs fixtures)